coalib/misc/Exceptions.py

```diff
@@ -15,6 +16,9 @@ def get_exitcode(exception, log_printer=None):
         exitcode = 0
     elif isinstance(exception, SystemExit):
         exitcode = exception.code
+    elif isinstance(exception, ContextualVersionConflict):
+        print("Version Conflict Error. Check 'pkg_resources'.")
```

Comment:
@rubik Yes. I was planning to do that. But, can you please check if this(the print message) is happening at all? I couldn't test it.